webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/uploading/UploadService.scala

```diff
@@ -163,6 +168,17 @@ class UploadService @Inject()(dataSourceRepository: DataSourceRepository,
       _ <- dataSourceService.assertDataDirWritable(reserveUploadInfo.organization)
       _ <- runningUploadMetadataStore.insert(redisKeyForFileCount(reserveUploadInfo.uploadId),
                                              String.valueOf(reserveUploadInfo.totalFileCount))
+      _ <- Fox.runOptional(reserveUploadInfo.totalFileSizeInBytes) { fileSize =>
+        Fox.combined(
+          List(
+            runningUploadMetadataStore.insert(redisKeyForTotalFileSizeInBytes(reserveUploadInfo.uploadId),
+                                              String.valueOf(fileSize)),
+            runningUploadMetadataStore
+              .insert(redisKeyForCurrentUploadedTotalFileSizeInBytes(reserveUploadInfo.uploadId), "0")
+          ))
+      }
+      _ <- runningUploadMetadataStore.insert(redisKeyForCurrentUploadedTotalFileSizeInBytes(reserveUploadInfo.uploadId),
```

Comment:
didn’t you just insert something at this key in the line above, inside of the runOptional?